### ISSUE TITLE
Color schemes should be configurable

### ISSUE THREAD

**Issue by mitya57** [retext-project/retext#62]
Light colours are tiring, but a system-wide dark colour scheme (like Solarized Dark) doesn't play well with some apps and in some situations. So ability to change the bacrground and font colours in config would be a fine convenient.

Reported by: tablethater

Original Ticket: [retext/tickets/62](https://sourceforge.net/p/retext/tickets/62)

**Comment by blonchkman** [comment]
Hello, Dmitri

Using ReText v5.2.1 on Ubuntu 15.10 (French locale) and Mate desktop, i'm unable to set a working color scheme file for dark bg.  
Here is my actual color scheme content:
```
htmlTags = aqua
htmlStrings = lime
htmlSymbols = yellow
htmlComments = gray
markdownLinks = fushia
blockquotes = silver
```

No more success by replacing named colors with `#rrggbb` codes. 
No more success when adding `[ColorScheme]` on top. 
No more success when adding these line in main config instead of a separate color scheme file.

Now, i've set a simple qtstyle css in order to revert desktop dark to ReText light bg.
Do i made something wrong or is it a known problem ?

Kind regards

**Comment by mitya57** [comment]
@blonchkman The documentation is for version 5.3, in 5.2 the color scheme settings were stored not in a configuration file, but in a separate file which you needed to link to the settings using a `colorSchemeFile` configuration option.

**Comment by blonchkman** [comment]
@mitya57  i apologize for confusion. I've try both ways carefully avoiding mixing them. Putting color settings in a separate file declared in `colorSchemeFile` option do not seams to work for me. ReText launched from the CLI doesn't show any error.

Content of colorscheme.conf:
```
htmlTags = aqua
htmlStrings = lime
htmlSymbols = yellow
htmlComments = gray
markdownLinks = fushia
blockquotes = silver
```
Content of ReText.conf
```
appStyleSheet=/home/toto/.config/ReText project/ReText-light.css
colorSchemeFile=/home/toto/.config/ReText project/colorscheme.conf
...
styleSheet=/home/toto/.config/ReText project/render.css
```

**Comment by mitya57** [comment]
@blonchkman Try lower-case names for the options (i.e. `htmltags`) as described in [ReText 5.2 configuration.md](https://github.com/retext-project/retext/blob/retext-5.2/configuration.md#color-scheme-files).

In any case, I recommend you to upgrade to the latest stable release (5.3.1) which is now available in Ubuntu 16.04 LTS.

**Comment by blonchkman** [comment]
@mitya57 - i apologize as it was the wrong case syntax. works like a charm after correction. Migration to 16.04 is planed so i wont boring you any more with old stuff bugs ;-)  

Thank you very much for all your help
Kind regards
